Example/DisplayController.swift

```diff
@@ -26,6 +27,10 @@ class DisplayController: UIViewController {
                 self.displayAlert(title: NSLocalizedString("error_title", comment: ""), message: NSLocalizedString("error_miniapp_download_message", comment: ""), dismissController: true)
                 print("Errored: ", error.localizedDescription)
             }
-        }
+        }, messageInterface: self)
+    }
+
+    func getUniqueId() -> String {
+        return ""
```

Comment:
shouldn't we return some dummy value ? Or make it nullable ?

Reply:
Yeah, we should make it return some value. Does iOS have a unique ID (either unique to the device or App) you can use here?

Reply:
Done